README.md

```diff
@@ -324,6 +324,96 @@ xsite:
       port: 7200
 ```
 
+Next the (local) port 7900 must be mapped to the port defined in the configuration (7200). On Docker for example by using `-p 7200:7900`
+
+Note: Caches won't be created automatically on the other side. So these must be created manually with the same name.
+
+So create a new cache either using the console (http://localhost:11222) or curl (or similar tool)
+`curl -XPOST -u <user>:<password> -H "Content-Type: application/xml" -d "@<path/to/cache.xml>" http://localhost:11222/rest/v2/caches/xsite-cache`
+  
+LON-Site:
+```
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsite-cache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC"/>
+      </backups>
+    </replicated-cache>
+  </cache-container>
+</infinispan>
+```
+
+NYC-Site:
+```
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsite-cache" statistics="true">
+      <backups>
+        <backup site="LON" strategy="SYNC"/>
```

Comment:
Also include "NYC" as a backup location.

Reply:
not needed :)